docs/odcs.rst

```diff
@@ -46,46 +50,49 @@ information for requesting a compose to be created by ODCS::
     compose:
       # Optional, default False. Inherit compose used for building parent image
       inherit: true
-      # Optional, default "tag".
-      type: tag
-      # Required for "tag" type. Must contain at least one item.
+      # Required for ODCS Koji "tag" type usage. Must contain at least one item.
       packages:
       - package-1-name # This is a package name, not an NVR.
       - package-2-name
-      # Optional. Default value is configured in OSBS environment
-      sigkeys:
-      - AB123456
+      # Optional. Default and possible values are configured in OSBS environment
+      # which will be a mapping of intent -> list of signing keys
+      #     release -> my-release-key
+      #     beta -> my-beta-key, my-release-key
+      #     unsigned -> <empty>
+      intent: release
```

Comment:
Do we want to use `intent` as the buildContainer parameter instead of `odcs_signed`?

Reply:
Similarly, should the intent appear in the Koji build metadata?

Reply:
This probably makes more sense at this point. Do we want `odcs_intent`? Or just `intent`?
Reason being that if ODCS integration is not enabled, `intent` is meaningless.

Reply:
Or `signing_intent` to make the key more self-documenting?

Reply:
+1 for `signing_intent` as the key in yaml file
Maybe the same for cli/api (`signing-intent/signing_intent`) parameter to avoid confusion?

Reply:
Added `signing_intent`.